api/application/client_test.go

```diff
@@ -1121,3 +1121,61 @@ func (s *applicationSuite) TestResolveUnitErrorsAll(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(called, jc.IsTrue)
 }
+
+func (s *applicationSuite) TestScaleApplication(c *gc.C) {
```

Comment:
Please also add a test to check when an error is returned from the call as well as an error returned as part of the result...

Reply:
ok